tests/evm/test_jumpi.py

```diff
@@ -0,0 +1,133 @@
+from zkevm_specs.evm import (
+    main, Opcode, ExecutionResult, CoreState, CallState, Step,
+    Tables, RWTableTag,FixedTableTag
+)
+from zkevm_specs.util import (linear_combine, keccak256, fp_inv)
+from utils import assign_bytecode_table
+
+
+# TODO: Parametrize r, table
+def test_jumpi_nonzero_cond():
+    r = 1
+    bytecode_source = '608060406007575b00'
```

Comment:
```suggestion
    # Jumps to PC=7 because the condition (40) is nonzero.
    # PUSH1 80 PUSH1 40 PUSH1 07 JUMPI JUMPDEST STOP
    bytecode_source = '608060406007575b00'
```